drivers/sdhc/Kconfig.emmc

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+config EMMC_HOST
+	bool "EMMC host controller driver support"
+	select SDHC_SUPPORTS_NATIVE_MODE
+	select EVENTS
+	default y
+	depends on DT_HAS_EMMC_HOST_ENABLED
+	help
+	  EMMC host controller driver support.
+
+if EMMC_HOST
+config EMMC_HOST_INTR
+	bool "EMMC host controller interrupt mode"
+	default y
+	help
+	  EMMC host controller interrupt mode support.
+
+config EMMC_HOST_DMA
+	bool "EMMC host controller DMA mode"
+	select DCACHE
+	help
+	  EMMC host controller DMA mode support.
+
+config EMMC_HOST_ADMA
+	bool "EMMC host controller ADMA mode"
+	depends on EMMC_HOST_DMA
+	help
+	  EMMC host controller ADMA mode support.
+
+config EMMC_HOST_ADMA_DESC_SIZE
+	int "Host controller ADMA Descriptor size"
+	depends on EMMC_HOST_DMA
+	default 32
+	help
+	  EMMC host controller ADMA Descriptor size.
+
+config EMMC_HOST_AUTO_STOP
+	bool "Auto stop command mode"
+	default y
+	help
+	  Auto stop command mode support.
+
+config EMMC_HOST_BLOCK_GAP
+	bool "Block gap mode"
+	depends on EMMC_HOST_DMA
+	help
+	  Block gap mode support.
+
+config EMMC_HOST_TUNING
+	bool "EMMC re-tuning support"
+	help
+	  EMMC re-tuing support.
+
+module = EMMC_HOST
```

Comment:
Why are additional logging Kconfigs being added here? `CONFIG_SDHC_LOG_LEVEL` should be used to set the log level in this driver

Reply:
Please remove these statements here that source `subsys/logging/Kconfig.template.log_config`